sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/parameterized_command.py

```diff
@@ -40,3 +41,4 @@ class ParameterizedCommandSchema(PathAwareSchema):
     )
     resources = NestedField(JobResourceConfigurationSchema)
     distribution = DistributionField()
+    queue_settings = NestedField(QueueSettingsSchema)
```

Comment:
Also need to do this for `CommandSchema` in `_schema/pipeline/component_job.py`

Reply:
`CommandSchema` in `_schema/pipeline/component_job.py` inherits from `ParameterizedCommandSchema` which already has `queue_settings` defined. Is it still expected to be added in that file?

Reply:
I'm also confused for this, but during my test on your branch, the `queue_settings` keeps `None`; after I add it in `CommandSchema`, everything goes fine. Not sure if it is something magic inner `marshmallow`.

Reply:
Also, you can merge this PR without the above change. I will open a PR for Pipelines after this merged.